MicrosoftTeams.schema.json

```diff
@@ -518,5 +455,190 @@
+        "actionParamCommand": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "description": "Id of the command.",
+                    "maxLength": 64
+                },
+                "title": {
+                    "type": "string",
+                    "description": "Title of the command.",
+                    "maxLength": 32
+                },
+                "description": {
+                    "type": "string",
+                    "description": "Description of the command.",
+                    "maxLength": 128
+                },
+                "type": {
+                    "type": "string",
+                    "const": "action",
+                    "description": "Type of the command"
+                },
+                "fetchTask": {
+                    "type": "boolean",
+                    "const": false
+                },
+                "parameters": {
+                    "type": "array",
+                    "maxItems": 5,
+                    "minItems": 1,
+                    "items": {
+                        "type": "object",
+                        "additionalProperties": false,
+                        "properties": {
+                            "name": {
+                                "type": "string",
+                                "description": "Name of the parameter.",
+                                "maxLength": 64
+                            },
+                            "inputType": {
+                                "type": "string",
+                                "enum": [
+                                    "text",
+                                    "textarea",
+                                    "number",
+                                    "date",
+                                    "time",
+                                    "toggle"
+                                ],
+                                "default": "text",
+                                "description": "Type of the parameter"
+                            },
+                            "title": {
+                                "type": "string",
+                                "description": "Title of the parameter.",
+                                "maxLength": 32
+                            },
+                            "description": {
+                                "type": "string",
+                                "description": "Description of the parameter.",
+                                "maxLength": 128
+                            }
+                        },
+                        "required": [
+                            "name",
+                            "title"
+                        ]
+                    }
+                }
+            },
+            "required": [
+                "type",
+                "parameters"
```

Comment:
Same as before, id and title should be required too

Reply:
yes